components/games/openttd/openttd.license

```diff
@@ -0,0 +1,343 @@
+This is the license which applies to OpenTTD with the exception of some
```

Comment:
Isn't this license already distributes with sources?  If so, then I suggest to use that one instead of creating a copy here.  To do so you would need to set `COMPONENT_LICENSE` accordingly.